chain-core/src/init/mod.rs

```diff
@@ -16,3 +18,9 @@ pub const MAX_COIN: u64 = 100_000_000_000__0000_0000;
 pub const MAX_COIN_DECIMALS: u64 = 1_0000_0000;
 /// 100 billion
 pub const MAX_COIN_UNITS: i64 = 100_000_000_000;
+
+#[cfg(feature = "testnet")]
+pub const CURRENT_NETWORK: network::Network = network::Network::Testnet;
```

Comment:
no need for these

Reply:
ok, i'll remove this feature.
so our binary will be one, networkid will work as runtime argument.